tests/test_mp_full.py

```diff
@@ -0,0 +1,95 @@
+"""
+multiproc full tests.
+"""
+
+import multiprocessing
+import platform
+import pytest
+import time
+
+import six
+import wandb
+
+
+def train(add_val):
+    time.sleep(1)
+    wandb.log(dict(mystep=1, val=2 + add_val))
+    wandb.log(dict(mystep=2, val=8 + add_val))
+    wandb.log(dict(mystep=3, val=3 + add_val))
+    wandb.log(dict(val2=4 + add_val))
+    wandb.log(dict(val2=1 + add_val))
+    time.sleep(1)
+
+
+def test_multiproc_default(live_mock_server, parse_ctx):
+    run = wandb.init()
+    train(0)
+    run.finish()
+
+    ctx_util = parse_ctx(live_mock_server.get_ctx())
+
+    summary = ctx_util.summary
+    s = {k: v for k, v in dict(summary).items() if not k.startswith("_")}
+    assert dict(val=3, val2=1, mystep=3) == s
```

Comment:
should we convert this check into a function? that way we can just call `check_summary` whenever we want to check the summary of a test using the train function?

Reply:
agreed, will do in next set of mp test changes.